src/SuperDumpService/Views/Home/_Threads.cshtml

```diff
@@ -141,12 +176,37 @@
 												<td class="nobreak stackmethodname">
 													<span class="@tagclasses">@modulename</span>
 													@if (frame.SourceInfo != null) {
-														<span class="sourceinfo">@frame.SourceInfo.File:@frame.SourceInfo.Line</span>
+														<span class="sourceinfo">@frame.SourceInfo.FileName():@frame.SourceInfo.Line</span>
 													}
 												</td>
 											}
-										</tr>
-									}
+                                            <td class="nobreak">
+                                                @if((frame as SuperDumpModels.SDCDCombinedStackFrame)?.HasStackInfo() ?? false) {
+                                                    int lines = 0;
+                                                    string stackHtml = "";
+                                                    if (((SuperDumpModels.SDCDCombinedStackFrame)frame).Args.Count > 0) {
+                                                        stackHtml += "<h4>Arguments</h4><br/><table border=\"1px\" width=\"450px\"><thead><td><b>&nbsp;Name&nbsp;</b></td><td><b>&nbsp;Value&nbsp</b></td></thead>";
```

Comment:
I'm sure we can find a better way to format create this string with a Razor template.